src/components/pages/enterprise/training/training.jsx

```diff
@@ -3,39 +3,80 @@ import React from 'react';
 
-const isovalent = {
-  title: 'Getting started with Cilium',
-  description:
-    'Quickly get started with Cilium. Read the documentation or use our interactive tutorial in a live environment.',
-  buttonText: 'Coming soon',
+const logos = {
+  acend: AcendLogo,
+  isovalent: IsovalentLogo,
 };
+
+const items = [
+  {
+    logoName: 'acend',
+    title: 'Cilium Basics',
+    description:
+      'This one-day training in English or German provides a practical and clear introduction to Cilium, an eBFP-based networking, observability and security stack for cloud and on-premise infrastructure.',
+    buttonText: 'Learn more',
+    buttonLink: 'https://www.acend.ch/en/trainings/cilium/',
+    buttonTarget: '_blank',
+  },
+  {
+    logoName: 'isovalent',
+    title: 'Getting started with Cilium',
+    description:
+      'Quickly get started with Cilium. Read the documentation or use our interactive tutorial in a live environment.',
+    buttonText: 'Coming soon',
+    buttonDisabled: true,
+  },
+];
+
 const Training = () => (
   <section className="py-10 md:py-20 lg:py-28">
-    <Container className="flex flex-col space-y-6 md:space-y-10 xl:flex-row  xl:items-center xl:space-y-0 xl:space-x-28">
-      <div className="xl:max-w-[384px]">
-        <Heading tag="h2">{title}</Heading>
-        <p
-          className="with-link-primary mt-5 text-lg"
-          dangerouslySetInnerHTML={{ __html: description }}
-        />
-      </div>
-      <div className="flex-grow bg-grid bg-cover bg-center bg-no-repeat pt-9 pb-11 sm:bg-contain md:pb-16 lg:pb-20">
-        <div className="relative mx-auto flex max-w-[384px] flex-col rounded-xl bg-white px-8 pb-8 pt-11 text-center shadow-card">
-          <IsovalentLogo className="h-10 w-auto" />
-          <Heading className="mt-10" size="2xs" tag="h3">
-            {isovalent.title}
-          </Heading>
-          <p className="mt-2.5">{isovalent.description}</p>
-          <Button className="mt-7 self-center" theme="primary-1" disabled>
-            {isovalent.buttonText}
-          </Button>
-        </div>
+    <Container className="">
+      <Heading className="text-center" tag="h2">
+        {title}
+      </Heading>
+      <p
+        className="with-link-primary mx-auto mt-4 max-w-[592px] text-center text-lg"
+        dangerouslySetInnerHTML={{ __html: description }}
+      />
+      <div className="mt-12 flex flex-col space-y-8 md:flex-row md:justify-between md:space-y-0 md:space-x-8">
+        {items.map(
+          (
+            { logoName, title, description, buttonText, buttonLink, buttonTarget, buttonDisabled },
+            index
+          ) => {
+            const Logo = logos[logoName];
+
+            return (
+              <div
+                className="relative mx-auto flex min-h-[320px] max-w-[592px] basis-1/2 flex-col rounded-xl bg-white p-4 shadow-card sm:p-6 md:min-h-[400px] md:p-8 lg:min-h-[350px]"
```

Comment:
I think we could get rid of min-h and set margin-bottom for the text below.

```suggestion
                className="relative mx-auto flex max-w-[592px] basis-1/2 flex-col rounded-xl bg-white p-4 shadow-card sm:p-6 md:p-8"
```